### DISCUSSION TITLE
Issue Forms does not persist header types added

### DISCUSSION THREAD

**Discussion by wsmelton** [community/community#4599] [Projects and Issues]
If I add just the header as a markdown type it is not being persisted after I submit it.

Steps:

1. Add a markdown type
2. Set the attributes > value to just : `"## Header Title"`
3. Submit an issue
4. Rendered issue is not including the header text, even when I edit it does not show up.

**Comment by SirSpidey** [comment]
Pulling together related requests for renderable Markdown:https://github.com/orgs/community/discussions/63402#discussioncomment-10341167
